File: app/src/main/java/buggy/plandroid/com/plandroidbugs/UserList.java
package buggy.plandroid.com.plandroidbugs;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.ArrayList;
import java.util.List;

@JsonIgnoreProperties(ignoreUnknown = true)
public class UserList {

    @JsonProperty("data") public List<UserWire> userWires = new ArrayList<>();
}

File: app/src/main/java/buggy/plandroid/com/plandroidbugs/UserList.kt
package buggy.plandroid.com.plandroidbugs

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class UserList(
    @JsonProperty("data")
    val userWires: List<UserWire> = ArrayList()
)